packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -87,37 +88,31 @@ export class MarketDB extends DerivedDB {
     }
   }
 
-  syncOrderBooks = async (syncing: boolean): Promise<void> => {
+  syncOrderBooks = async (): Promise<void> => {
     const highestSyncedBlockNumber = await this.syncStatus.getHighestSyncBlock(this.dbName);
     const documents = [];
 
-    const currentOrderLogs = await this.stateDB.CurrentOrders.where("blockNumber").aboveOrEqual(highestSyncedBlockNumber).toArray();
+    const currentZeroXOrders = await this.stateDB.ZeroXOrders.toArray();
 
-    if (currentOrderLogs.length < 1) return;
+    if (currentZeroXOrders.length < 1) return;
 
-    const marketIds: string[] = _.uniq(_.map(currentOrderLogs, 'market')) as string[];
-    const highestBlockNumber: number = _.max(_.map(currentOrderLogs, 'blockNumber')) as number;
+    const marketIds: string[] = _.uniq(_.map(currentZeroXOrders, 'market')) as string[];
     const marketsData = await this.stateDB.Markets.where("market").anyOf(marketIds).toArray();
-
-    const marketDataById = _.keyBy(marketsData, 'market');
-
     const reportingFeeDivisor = await this.augur.contracts.universe.getReportingFeeDivisor_();
     // TODO Get ETH -> DAI price via uniswap when we integrate that as an oracle
     const ETHInAttoDAI = new BigNumber(200).multipliedBy(10**18);
 
+    const marketDataById = _.keyBy(marketsData, 'market');
+
     for (const marketId of marketIds) {
       const doc = await this.getOrderBookData(this.augur, marketId, marketDataById[marketId], reportingFeeDivisor, ETHInAttoDAI);
       // This is needed to make rollbacks work properly
-      doc['blockNumber'] = highestBlockNumber;
+      doc['blockNumber'] = highestSyncedBlockNumber;
```

Comment:
Do we even need this anymore?

Reply:
I don't think so -- let me double check, i think i just missed that in cleanup

Reply:
More on this. It appears this function now only does a full one time sync with no update capability. How do we handle updating liquidity as new orders come in? Previously we would get the latest order states from the currentOrders DB which would handle decreases or complete fills to decrease a markets liquidity as well as any new orders. In this code I think we'd end up updating the world on every single call to this. Is this now only used for bulk sync?

Reply:
I thought i had this wired up to run every-time the zeroXOrders table gets updated
```this.augur.events.subscribe('DerivedDB:updated:ZeroXOrders', this.syncOrderBooks);```


Also i left blockNumber logic in b/c of the comment below it, wasn't sure if i should remove it
```
// This is needed to make rollbacks work properly
```